src/chain_spec.rs

```diff
@@ -1084,6 +1088,10 @@ fn development_genesis(
 		parachain_system: Default::default(),
 		treasury: Default::default(),
 		interest_accrual: Default::default(),
+		base_fee: Default::default(),
+		evm_chain_id: development_runtime::EVMChainIdConfig { chain_id: 999_999 },
```

Comment:
Generally test networks have different IDs in the Ethereum ecosystem. That would mean even Catalyst should have a unique ID from the production Centrifuge chain. It is a 64-bit ID, so there is lots of room in the namespace.

I think it's better that any code we write (even tests) be chain-id-agnostic, rather than breaking the convention and reusing production chain IDs for local networks